src/indexer/app.rs

```diff
@@ -0,0 +1,13 @@
+extern crate clap;
+use self::clap::{Arg, App, SubCommand};
+
+pub fn make() -> App<'static, 'static> {
+    App::new("golden-retriever-indexer")
+        .version("0.0.1")
```

Comment:
Try to pull the version number from `Cargo.toml`. Not sure how but should be support for this somewhere.